pyomo/contrib/mindtpy/outer_approximation.py

```diff
@@ -141,15 +138,7 @@ def objective_reformulation(self):
         # TODO: The logic here is too complicated, can we simplify it?
```

Comment:
Is this comment still relevant?

Reply:
I have simplified the arguments of this function. The logics are necessary and are still implemented inside the function. We can remove this comment.

Reply:
Great, please remove it